meta/main.yml

```diff
@@ -1,15 +1,18 @@
 ---
 galaxy_info:
   author: groks
-  description: "enable the systemd journal and limit it's appetite for disk space"
+  description: "manage systemd journald configuration"
   license: BSD
-  min_ansible_version: 1.3
+  min_ansible_version: 2.1
```

Comment:
https://www.ansible.com/blog/loop-plays-past-present-future says that the loop: keyword was introduced in 2.5. I think we should set min version to 2.5

Reply:
yeah, I forgot about the loop :) fixed it in rebase